src/components/TimeBadge.vue

```diff
@@ -16,103 +16,109 @@ export default {
   props: {
     starttime: {
       type: String,
-      default: '1970-01-01T00:00:00.000Z',
+      default: '1970-01-01T00:00:00.000Z'
     },
     endtime: {
       type: String,
-      default: '1970-01-01T00:00:00.000Z',
+      default: '1970-01-01T00:00:00.000Z'
     },
     text: {
       type: String,
-      default: '0s',
+      default: '0s'
     },
     variant: {
       type: String,
       default: 'info'
     },
     interval: {
       type: Number,
-      default: 60000,
+      default: 60000
     }
   },
   mounted: function() {
-    this.onBadgeUpdate();
+    this.onBadgeUpdate()
   },
-  data: function () {
+  data: function() {
     return {
       disp: this.text,
       mutableVariant: this.variant
     }
   },
   methods: {
     formatDurationShort(duration) {
-      let timeText = '';
+      let timeText = ''
       if (duration.days()) {
-        timeText += `${duration.days()}d ${duration.hours()}h ${duration.minutes()}m ${duration.seconds()}s`;
+        timeText += `${duration.days()}d ${duration.hours()}h ${duration.minutes()}m ${duration.seconds()}s`
       } else if (duration.hours()) {
-        timeText += `${duration.hours()}h ${duration.minutes()}m ${duration.seconds()}s`;
+        timeText += `${duration.hours()}h ${duration.minutes()}m ${duration.seconds()}s`
       } else if (duration.minutes()) {
-        timeText += `${duration.minutes()}m ${duration.seconds()}s`;
+        timeText += `${duration.minutes()}m ${duration.seconds()}s`
       } else {
-        timeText += `${duration.seconds()}s`;
+        timeText += `${duration.seconds()}s`
       }
-      return timeText;
+      return timeText
     },
     formatTimer(diff) {
-      let timeLeft = diff;
-      const stringArray = [];
+      let timeLeft = diff
+      const stringArray = []
 
-      [[86400000, 'd'], [3600000, 'h'], [60000, 'm'], [1000, 's']]
-        .forEach(([unit, suffix]) => {
-          const time = Math.floor(timeLeft / unit);
-          const first = stringArray.length === 0;
+      ;[[86400000, 'd'], [3600000, 'h'], [60000, 'm'], [1000, 's']].forEach(
```

Comment:
Semicolon at the beginning of a line? Some weird style changes are happening here.